WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Categories/Add Category/AddEditProductCategoryViewController.swift

```diff
@@ -176,6 +181,15 @@ extension AddEditProductCategoryViewController: UITableViewDelegate {
             cell?.resignFirstResponder()
         }
     }
+
+    /// Focus on Title Category Text Field
+    ///
+    private func focusTitleTextField() {
```

Comment:
This has a very similar implementation with `titleCategoryTextFieldResignFirstResponder`. Can we combine the two methods to reuse most of the logic?

```suggestion
    private func updateTitleTextField(shouldFocus: Bool) {
        // Depending on `shouldFocus`, we can make the cell become or resign first responder.
    }
```

Reply:
Thanks, Huong! Updated here 0690b4848d1f45251b98d34d2168e40adb84932a